.github/workflows/versionincrement.yml

```diff
@@ -0,0 +1,89 @@
+---
+name: Version Increment
```

Comment:
This increments plugin versions? Let's call the file `increment-plugin-versions.yml`, and the task "Increment Plugin Versions"?

Reply:
Updated @dblock 
`name: Increment Plugin's Version` 
`FileName: increment-plugin-versions.yml`